platforms-wg/glossary/_index.md

```diff
@@ -1,43 +1,43 @@
+Dependiendo del alcance y la audiencia de una plataforma, a veces se puede referir a ella como una "Plataforma de Desarrollo", una "Plataforma de Desarrollo Interno (IDP)", una "Plataforma de Entrega", una "Plataforma de Aplicaciones" o incluso una "Plataforma en la Nube". El término "Plataforma como Servicio (PaaS)" también se usa a menudo para describir plataformas que se compran o adoptan desde fuera de una organización, proporcionando una solución de plataforma más gestionada pero a menudo menos personalizable.
 
-## Platform engineering
-The design, construction, operation, and evolution of a platform. One way to view the practice is as an empathy-driven approach towards sociotechnical organizational design<sup><a href="https://hazelweakly.me/talks/qcon-sf-2023/slides#22">1</a></sup>. In this light, it is a continuous process by which an organization learns how and where to invest and to make strategic business bets internally, rather than just externally.
+## Ingeniería de Plataforma (Platform engineering)
+El diseño, construcción, operación y evolución de una plataforma. Una forma de verlo en la práctica es un enfoque impulsado por la empatía hacia el diseño organizacional sociotécnico<sup><a href="https://hazelweakly.me/talks/qcon-sf-2023/slides#22">1</a></sup>. En este sentido, es un proceso continuo por el cual una organización aprende cómo y dónde invertir y hacer apuestas comerciales estratégicas internamente, en lugar de solo externamente.
 
-## Platform team
-The people responsible for building and managing the platform(s). Platform team members include **platform engineers**, who focus on building the tool and capabilities that comprise the platform experiences. It may include dedicated **platform product manager** roles that focus on meeting internal customer needs while also supporting the organization’s broader strategic goals. As the platform evolves, other specialist roles, such as operators, QA analysts, UI/UX designers, technical writers, and developer advocates, may be added to platform teams.
+## Equipo de Plataforma
+Las personas responsables de construir y gestionar la(s) plataforma(s). Los miembros del equipo de plataforma incluyen **ingenieros de plataforma**, que se centran en la construcción de la herramienta y las capacidades que componen las experiencias de la plataforma. Puede incluir roles de **gestor de producto de plataforma** dedicados que se centran en satisfacer las necesidades de los clientes internos y al mismo tiempo apoyan los objetivos estratégicos más amplios de la organización. A medida que la plataforma evoluciona, otros roles especializados, como operadores, analistas de QA, diseñadores UI/UX, redactores técnicos y developer advocates, pueden ser añadidos a los equipos de plataforma.
```

Comment:
"que se centran en la construcción de la **herramientas**", this is plural instead of singular.

Reply:
"who focus on building the tool", I think this is singular otherwise it would be "who focus on building the tools".

Reply:
LGTM :)